dev/ci/user-overlays/README.md

```diff
@@ -1,38 +1,72 @@
 # Add overlays for your pull requests in this directory
 
-When your pull request breaks an external project we test in our CI and you
-have prepared a branch with the fix, you can add an "overlay" to your pull
-request to test it with the adapted version of the external project.
+_Overlays_ let you test pull requests that break the base version of
+external projects by applying PRs of the external project during CI
+testing (1 PR per broken external project).  Once Coq CI's test of the
+external project passes, the Coq PR can be merged, after which coqbot
+sends a request to the external projects to merge their PRs.
```

Comment:
This is incorrect, coqbot only posts "please take care of overlays" in the merged coq PR. It's up to the assignee to tell 3rd parties to merge the overlay PRs, and up to the 3rd party devs to merge in a timely fashion.